### PR TITLE
Logo Css and Verb fix

### PR DESCRIPTION
**Pull request by lcurtis4**
## Changes 
adding css to the logo 
correcting differences in verbs used 
## Steps to test 
check all html files to see verbs match 
run serve 
see that logo moves to center of header on all pages 
see that all categories titles match

### CODE REVIEW

File: india.html [hunk 1]
```diff
@@ -32,23 +32,15 @@ <h2 class="label">Cities To Visit</h2>
                     </ul>
                 </section>
                 <section class="lists__landmarks">
-<<<<<<< HEAD
-                    <h2>Famous Landmarks</h2>
-=======
                     <h2 class="label">Landmarks To Visit</h2>
```
Comment (Chris-Denmark) [inline india.html]: Leaving a suggestion for changing this to be uniform with the other pages.

**Comment by Chris-Denmark** [comment]
👍

**Comment by cslunetta** [comment]
👍

**Comment by jicarter** [comment]
👍

**Pull request merged**